content/blogposts/2022/sourcegraph-4.0.md

```diff
@@ -0,0 +1,190 @@
+
+Performance across the platform saw a boost and search results are now returned faster than before by using the latest in React, lazy loading, and priority rendering. 
+
+![UI Improvements](https://storage.googleapis.com/sourcegraph-assets/blog/release-post/4.0/Search%20UI%20Improvements.png "UI Improvements")
+
+<Badge text="Code Insights" color="green" size="small" onClick={() => window.open("/code-insights", "_self")} />
+
+#### Understand usage and search structure with high-level aggregations of search results
+It’s now easier for you to get high-level knowledge from the codebase and answer questions with aggregations of search results powered by Code Insights. In the chart that appears on the search results page, you can choose to group your search results by location (repo or file), author, or arbitrary capture group pattern.
+
+The new aggregations answer:
+- How many different versions of a library or package are present in your code, and which is most common? 
+- Which files have the majority of these search results?
+- Which repositories is this library used in most?
+- Which library components are used most? By who?
+- What are all the possible arguments we pass to this function? 
+
+Here are some [example search aggregations to get started with](https://docs.sourcegraph.com/code_insights/references/search_aggregations_use_cases)
+
+<Video 
+  source={{
+    webm: 'blog/release-post/4.0/Search_Aggregations_4.0',
+    mp4: 'blog/release-post/4.0/Search_Aggregations_4.0'
+  }}
+  loop={true}
+  title="High-level aggregations"
+  caption="High-level aggregations of search results"
+/>
+
+
+[Docs](https://docs.sourcegraph.com/code_insights/explanations/search_results_aggregations)
+
+#### Configure precise code navigation for 9 languages in a matter of minutes with auto-indexing
+Code navigation allows you to quickly jump to symbol definitions, find references, dependencies, and more, so you can spend more time shipping code. Precise code navigation is the most powerful version of code navigation and it works cross-repository. Auto-indexing allows you to set up precise code navigation for any chosen repositories with the click of a button, and it makes precise code navigation work across all of your code, automatically. Auto-indexing is supported for 9 languages including Ruby, Rust, Go, Java, Scala, Kotlin, Python, TypeScript, JavaScript, and it is turned on by default for our Cloud customers.
+
+<Video 
+  source={{
+    webm: 'blog/release-post/4.0/ruby-precise',
+    mp4: 'blog/release-post/4.0/ruby-precise'
+  }}
+  loop={true}
+  title="Ruby precise code navigation"
+  caption="Precise code navigation with Ruby"
+/>
+
+[Docs](https://docs.sourcegraph.com/code_navigation/references/indexers)
+
+[Extensions]
+
+#### Your favorite extensions are now available by default 
```

Comment:
@lcolston08 - It's a bit confusing what is changing. What does default functionality mean? This paragraph jumps from idea to idea.

I think it'll help to follow the flow of the deprecation blog post.

Reply:
This has been updated if you could please review.

Reply:
Ok - this has been updated and I added the statement about the changes to the browser extensions. I also change the "Please not that this does not impact" to "This update does not impact..." since we use "Note:" in the statement above it.